boltdesignsystem/bolt
# WWW-105 PatternLab video clean up

## Jira

https://pegadigitalit.atlassian.net/browse/WWW-105

## Summary

Updated the existing PatternLab instance with the new version of the Brightcove video

## Details

1. Updated all the examples of videos being used within the PatternLab instance to the new Brigthcove video method
2. Added a "Deprecated" label on the Videos PL tab and added a deprecation message to the videos docs
3. Updated the build to output an isolated Brightcove CSS file for use within the Brigthcove Studio player

## How to test

Check instances of videos in the PL instance and confirm they are still working as expected
